src/cdk/drag-drop/drag-drop.md

```diff
@@ -1,213 +1,225 @@
-The `@angular/cdk/drag-drop` module provides you with a way to easily and declaratively create

```

Comment:
@crisbeto not sure why GitHub is picking up this diff incorrectly now, I've checked locally and the only changes are the additional API docs.
Have you seen this on other PRs?

Reply:
nvm, sorted it using vim.
Webstorm issue